### ISSUE TITLE
Rebuild button does nothing

### ISSUE THREAD

**Issue by galindro** [jenkinsci/pipeline-graph-view-plugin#361]
### Jenkins and plugins versions report

<details>
  <summary>Environment</summary>

  
  ```text
  Jenkins version: 2.440.1
  pipeline-graph-view-plugin version: 230.vcd408a_1308dc
  ```

</details>

### What Operating System are you using (both controller, and any agents involved in the problem)?

RHEL8

### Reproduction steps

Try to rebuild any pipeline via the Rebuild button.

![image](https://github.com/jenkinsci/pipeline-graph-view-plugin/assets/4622097/37622792-f5eb-4cc1-ad3d-497b083abc1b)

### Expected Results

Last pipeline execution should rebuild

### Actual Results

Nothing happens

**Comment by galindro** [comment]
Maybe related to this: https://github.com/jenkinsci/pipeline-graph-view-plugin/issues/330

**Comment by timja** [comment]
Yes, does yours have a parameter if so it's a duplicate? otherwise can you please provide any logs or more information like a sample pipeline?

**Comment by galindro** [comment]
Yes, it has parameters. So I guess it is a dup of https://github.com/jenkinsci/pipeline-graph-view-plugin/issues/330
